specification/http/src/main/scripts/org/kaazing/specification/http/rfc7230/connection.management/client.with.pipelining.must.not.retry.pipelining.immediately.after.failure/request.rpt

```diff
@@ -29,6 +29,7 @@ read "HTTP/1.1 200 OK\r\n"
 read "Content-Length: 8\r\n"
 read "\r\n"
 read "request1"
+closed
```

Comment:
That is exactly the issue in this bug. The server has two different responses on `http://localhost:8080/request2`, but the order is messed up on Ubuntu, the first answer goes to the second request from the client script. So, we need to make sure that the first connection is closed (and the server sent the response for the first request2) and then make the second request. If we will put the notify before the close, the close will be useless and the functionality will be the same as before the change.

I'm not sure what is the functionality of `read closed`, if it is equivalent to `closed`. I'll check it and add another comment here.